coralproject/cay
# Cay polishing 13062016

## What does this PR do?
- Removes the default inactive which was overriding the one on the form
- Cleans the form when loading the builder
## How do I test this PR?
- Forms should be correctly saved as inactive or active depending on the form builder switch.
- Go to create a new a form, save, click again on create form, it should be all blank slate.
